www/src/components/LayoutHeader/Header.js

```diff
@@ -66,7 +66,15 @@ const Header = ({location}) => (
                 fontSize: 16,
               },
               [media.lessThan('small')]: {
-                visibility: 'hidden',
+                // Visually hidden
+                position: 'absolute',
+                overflow: 'hidden',
+                clip: 'rect(0 0 0 0)',
+                height: 1,
+                width: 1,
+                margin: -1,
+                padding: 0,
+                border: 0,
```

Comment:
Why not `display: none` ?

Reply:
Due to the lack of alt attribute (see the other comment), we still want the site title to be read if the crawler is using mobile

Reply:
Gotcha. This is gross. 😛  But I think your logic is reasonable. The grossness is just browser stuff. 😁

Reply:
Gotcha. This is gross. 😛  But I think your logic is reasonable. The grossness is just browser stuff. 😁

Reply:
I might look into changing the markup slightly after this is merged to avoid this if you don't oppose it.